### PR TITLE
Update Docs around AzureConverter

### PR DESCRIPTION
**Pull request by tholor**
Preprocessing components page:
- Adds Azure + Markdown file converters
- Adjusts description of formats

Table QA Guide: 
- Link Azure Converter

### CODE REVIEW

**Comment by tholor** [comment]
@brandenchan I somehow could not access the preprocessing page in the vercel preview. Do you see why this might be broken?

File: docs/latest/components/preprocessing.mdx [hunk 1]
```diff
@@ -17,16 +18,25 @@ Check out our [preprocessing tutorial](/tutorials/train-dpr) if you'd like to st
...
 
-The DocumentStore expects its inputs to come in the following format.
-The sections below will show you all the tools you'll need to ready your data for storing.
+You can index `Document` objects or plain dictionaries to a Haystack `DocumentStore`. Therefore, the output of your preprocessing should comply to one of these two formats (we recommend `Document`, see [Primitives](/components/primitives) for more details): 
 
 ```python
+# Option 1: Native Haystack Documents
+docs = [
+    Document(
+        'content': DOCUMENT_TEXT_HERE,
```
Comment (brandenchan) [inline docs/latest/components/preprocessing.mdx]: This syntax doesn't look right to me - guessing you copied it from the dictionary example?

File: docs/latest/components/preprocessing.mdx [hunk 2]
```diff
@@ -17,16 +18,25 @@ Check out our [preprocessing tutorial](/tutorials/train-dpr) if you'd like to st
 
 ## Document Format
 
-The DocumentStore expects its inputs to come in the following format.
-The sections below will show you all the tools you'll need to ready your data for storing.
+You can index `Document` objects or plain dictionaries to a Haystack `DocumentStore`. Therefore, the output of your preprocessing should comply to one of these two formats (we recommend `Document`, see [Primitives](/components/primitives) for more details):
```
Comment (brandenchan) [inline docs/latest/components/preprocessing.mdx]: I think we should simplify the wording and punctuation here. My suggestion
"""
You can index `Document` objects (recommended) or plain dictionaries into a Haystack `DocumentStore`. 
The output of your preprocessing should comply with one of these two formats.
To learn more about the `Document` class see [Primitives](/components/primitives).
"""

File: docs/latest/guides/table_qa.mdx [hunk 3]
```diff
@@ -28,6 +28,8 @@ docs = [
 document_store.write_documents(docs)
 ```
 
+If you have your tables embedded in raw files (e.g. within a PDF), you can use the `AzureConverter` to parse them into the required format (see [File Conversion](/preprocessing#file-conversion)).
```
Comment (brandenchan) [inline docs/latest/guides/table_qa.mdx:31]: Visually I think this will be much nicer if we wrap it in a highlight box - see lines 6-10 of this file

**Review by brandenchan — CHANGES_REQUESTED** [review]

File: docs/latest/components/preprocessing.mdx (hunk 1 above)
Comment (tholor) [reply docs/latest/components/preprocessing.mdx]: Good catch! copy&paste bug :)

File: docs/latest/guides/table_qa.mdx (hunk 3 above)
Comment (tholor) [reply docs/latest/guides/table_qa.mdx:31]: I personally really don't like the current design of these highlight boxes. Their width doesn't fit the code boxes or paragraphs and the coloring looks quite distracting (especially in the dark mode): 

image

I would therefore keep it as regular text for now. Let's improve the design of these highlight boxes soon and then use them in more places.

File: docs/latest/components/preprocessing.mdx (hunk 2 above)
Comment (tholor) [reply docs/latest/components/preprocessing.mdx]: Done

File: docs/latest/components/preprocessing.mdx (hunk 1 above)
Comment (tholor) [reply docs/latest/components/preprocessing.mdx]: Fixed

**Review by brandenchan — APPROVED** [review]
